src/Component/MovieCard/MovieCards.js

```diff
@@ -1 +1,13 @@
-import './MovieCards.css'
\ No newline at end of file
+import './MovieCards.css'
+
+
+function MovieCards({ movie }){
```

Comment:
Are we destructuring movies later on in the project?

Reply:
Noticed that the Posters need their ratings underneath after I pulled.

Reply:
Good catch! I'll fix that tomorrow. Totally forgot. I was thinking I might wait for API to destructure but I might go ahead and start tomorrow when I fix the ratings if it's not gonna cause any issues.